### PR TITLE
e2e/network: skip new session affinity timeout tests for OVNKubernetes

### PR DESCRIPTION
**Pull request by dcbw**
Session affinity doesn't work yet in ovnkube; these tests were recently added so we need to ignore them too for now. We already have SDN-510 for implementing session affinity.

Fixes: 16abec0d471f3c

@danwinship @openshift/networking

### CODE REVIEW

**Comment by dcbw** [comment]
/retest

```
Running hack/verify-generated-bindata.sh:16: executing 'git diff --quiet /go/src/github.com/openshift/origin/test/extended/util/annotate/generated/' expecting success...
FAILURE after 0.042s: hack/verify-generated-bindata.sh:16: executing 'git diff --quiet /go/src/github.com/openshift/origin/test/extended/util/annotate/generated/' expecting success: the command returned the wrong error code
There was no output from the command.
There was no error output from the command.
[ERROR] hack/lib/cmd.sh:241: `return "${return_code}"` exited with status 1.
```

File: test/extended/util/annotate/rules.go [hunk 1]
```diff
@@ -204,6 +204,8 @@ var (
 			`\[sig-network\] Services should be able to switch session affinity for service with type clusterIP`,
 			`\[sig-network\] Services should have session affinity work for NodePort service`,
 			`\[sig-network\] Services should have session affinity work for service with type clusterIP`,
+			`\[sig-network\] Services should have session affinity timeout work for NodePort service`,
```
Comment (smarterclayton) [inline test/extended/util/annotate/rules.go:207]: Please link these comments to a bug or card that we expect to fix them

Comment (dcbw) [reply test/extended/util/annotate/rules.go:207]: @smarterclayton They already are: the same card as the section they are in, SDN-510. They are new tests added recently for affinity, and we already have a card for that.

**Comment by dcbw** [comment]
e2e-gcp failure is:

`[sig-instrumentation][Late] Alerts shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured [Suite:openshift/conformance/parallel]`

```
Apr 05 19:29:32.247 W ns/default pod/recycler-for-nfs-r92fd node/ci-op-2vm8f-w-c-6fn8d.c.openshift-gce-devel-ci.internal reason/FailedMount MountVolume.SetUp failed for volume "vol" : mount failed: exit status 32\nMounting command: systemd-run\nMounting arguments: --description=Kubernetes transient mount for /var/lib/kubelet/pods/97c4c10c-edc6-4e56-82d5-7f574e630fb5/volumes/kubernetes.io~nfs/vol --scope -- mount -t nfs 10.131.1.86:/exports /var/lib/kubelet/pods/97c4c10c-edc6-4e56-82d5-7f574e630fb5/volumes/kubernetes.io~nfs/vol\nOutput: Running scope as unit: run-r7e920bc4590a47d1966620a253deb4d2.scope\nmount.nfs: No route to host\n
Apr 05 19:30:35.527 I ns/openshift-kube-controller-manager namespace/openshift-kube-controller-manager reason/SecretUpdated Updated Secret/csr-signer -n openshift-kube-controller-manager because it changed (121 times)
Apr 05 19:31:28.261 I ns/openshift-etcd-operator deployment/etcd-operator reason/OperatorStatusChanged Status for clusteroperator/etcd changed: Degraded message changed from "NodeControllerDegraded: All master nodes are ready\nEtcdMembersDegraded: No unhealthy members found" to "NodeControllerDegraded: All master nodes are ready\nEtcdMembersDegraded: ci-op-2vm8f-m-2.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-0.c.openshift-gce-devel-ci.internal members are unhealthy,  members are unknown" (3 times)
Apr 05 19:31:28.279 W clusteroperator/etcd changed Available to False: EtcdMembers_No quorum: EtcdMembersAvailable: ci-op-2vm8f-m-1.c.openshift-gce-devel-ci.internal members are available,  have not started, ci-op-2vm8f-m-2.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-0.c.openshift-gce-devel-ci.internal are unhealthy
Apr 05 19:31:28.350 I ns/openshift-etcd-operator deployment/etcd-operator reason/UnhealthyEtcdMember unhealthy members: ci-op-2vm8f-m-1.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-2.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-0.c.openshift-gce-devel-ci.internal (43 times)
Apr 05 19:32:28.256 W clusteroperator/etcd changed Available to True: AsExpected: StaticPodsAvailable: 3 nodes are active; 3 nodes are at revision 2\nEtcdMembersAvailable: ci-op-2vm8f-m-1.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-2.c.openshift-gce-devel-ci.internal,ci-op-2vm8f-m-0.c.openshift-gce-devel-ci.internal members are available,  have not started,  are unhealthy,  are unknown
Apr 05 19:32:29.303 - 73s   I test="[sig-instrumentation][Late] Alerts shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured [Suite:openshift/conformance/parallel]" running
```

**Comment by dcbw** [comment]
@smarterclayton PTAL thanks; we have a card already for the newly skipped tests: SDN-510

**Comment by dcbw** [comment]
@smarterclayton @eparis the failed e2e-aws-ovn test is:

```
[sig-instrumentation]  Prometheus when installed on the cluster should start and expose a  secured proxy and unsecured metrics  [Suite:openshift/conformance/parallel] expand_less | 30s
-- | --
fail [github.com/openshift/origin/test/extended/prometheus/prometheus.go:280]: some services expose metrics over insecure channel Expected     <[]error \| len:9, cap:16>: [         {             s: "following target does not secure metrics endpoint: ovnkube-master",         },         {             s: "following target does not secure metrics endpoint: ovnkube-master",         },         {             s: "following target does not secure metrics endpoint: ovnkube-master",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },         {             s: "following target does not secure metrics endpoint: ovnkube-node",         },     ] to be empty                                  open stdout
```

which wasn't caught earlier because OVN is not a blocking test for most repos at this point.  3rd thing in the past two weeks (including the issue this PR fixes). I guess we should turn it on for more repos by default, with the understanding we (SDN team) will obviously jump on failures people find.

**Comment by juanluisvaladas** [comment]
@dcbw I was doing it for openshift-sdn which should get merged today.
https://github.com/openshift/cluster-network-operator/pull/574

 It's not a big deal to make it for OVN as well so I'll file a new bug for this and start working on it today.

**Comment by trozet** [comment]
> not gonna pass due to lack of secured metrics endpoints (which is being worked on by @juanluisvaladas )

@dcbw wont it also not pass because we are missing the ARP fixes in the usptream->downstream merge for ovn-k8s (np tests should fail I think)?

**Comment by dcbw** [comment]
> @dcbw wont it also not pass because we are missing the ARP fixes in the usptream->downstream merge for ovn-k8s (np tests should fail I think)?

Perhaps, but only e2e-gcp is blocking so this should be able to go in without e2e-aws-ovn passing.

**Comment by dcbw** [comment]
Test failures are filed as

https://bugzilla.redhat.com/show_bug.cgi?id=1822286
https://bugzilla.redhat.com/show_bug.cgi?id=1822289

**Pull request merged**